lib/registrant.rb

```diff
@@ -0,0 +1,19 @@
+class Registrant
+    attr_reader :name, :age, :permit, :license_data
+
+    def initialize(name, age, permit = false)
+    @name = name
+    @age = age
+    @permit = permit
+    @license_data = {written: false, license: false, renewed:false}
```

Comment:
Keep an eye on your indentation here - we want code that is inside of a method to be indented one level:

```
def initialize(name, age, permit = false)
  @name = name
  @age = age
  @permit = permit
  @license_data = {written: false, licence: false, renewed: false}
end
```